recipes/r-cerebroprepare/meta.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+    name: r-cerebroprepare
+    version: 1.0.0
+
+source:
+    url: https://github.com/romanhaa/Cerebro/archive/v1.0.tar.gz
+    sha256: f7bffeb1269abb7fc2769b7e261cb81666d1c4ca521630992a8cdd2407df80d6
+
+build:
+    number: 0
+    noarch: generic
+    rpaths:
+        - lib/R/lib/
+        - lib/
+
+requirements:
+    run:
+        - r-base
+        - bioconductor-biomart >=2.38.0
+        - r-dplyr >=0.7.8
+        - r-future.apply >=1.1.0
+        - r-httr >=1.4.0
+        - r-matrix >=1.2
+        - r-pbapply >=1.4
+        - r-readr >=1.3.1
+        - r-seurat >=2.3.0
+        - r-tidyr >=0.8.2
+    host:
+        - r-base
+        - bioconductor-biomart >=2.38.0
+        - r-dplyr >=0.7.8
+        - r-future.apply >=1.1.0
+        - r-httr >=1.4.0
+        - r-matrix >=1.2
+        - r-pbapply >=1.4
+        - r-readr >=1.3.1
+        - r-seurat >=2.3.0
+        - r-tidyr >=0.8.2
+
+test:
+    commands:
+        - $R -e "library('BiocManager'); BiocManager::install('romanhaa/cerebroPrepare')"
+
+about:
+    home: https://github.com/romanhaa/cerebroPrepare
+    license: MIT
+    summary: An R wrapper for cereproPrepare
+    description: |
+        An R wrapper for cerebroPrepare, the tool used convert data from Seurat objects
+        into cerebro objects, cabaple of being examined in the cerebro application.
+        See <https://github.com/romanhaa/Cerebro> for information regarding the application.
```

Comment:
I'm not sure how the `<https...>` will render.

Reply:
Actually, use `about: home:`, `about: dev_url:`, `about: doc_url` instead of links in description. In this case, it's pretty much redundant.